app/middlewares/errors.js

```diff
@@ -5,7 +5,8 @@ const DEFAULT_STATUS_CODE = 500;
 
 const statusCodes = {
   [errors.DATABASE_ERROR]: 503,
-  [errors.DEFAULT_ERROR]: 500
+  [errors.DEFAULT_ERROR]: 500,
+  [errors.AXIOS_ERROR]: 800
```

Comment:
what does mean 800 status code? I think that is better 503, what do you think?

Reply:
Hahaha actually, I was not sure what statusCode set. 
Thanks so much for the feedback! Fixed it!